Multisig/Logic/Web3AuthManager.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Web3AuthManager.swift
+//  Multisig
+//
+//  Created by Mouaz on 6/10/23.
+//  Copyright © 2023 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import CustomAuth
+class Web3AuthManager {
+    static let shared = Web3AuthManager()
+
+    private init() { }
+
+    //    func loginGoogle() async {
```

Comment:
what's with the commented out code? As an exception there should either be specific instructions what to uncomment / adjust or the whole block should not be committed at all.